action/protocol/staking/handler_stake_migrate_test.go

```diff
@@ -284,7 +310,7 @@ func TestHandleStakeMigrate(t *testing.T) {
 		r.Equal(uint64(iotextypes.ReceiptStatus_Success), receipts[0].Status)
 		// gas = instrinsic  + contract call
 		instriGas, _ := act.IntrinsicGas()
-		r.Equal(receipt.GasConsumed+instriGas, receipts[0].GasConsumed)
+		r.Equal(instriGas, receipts[0].GasConsumed)
```

Comment:
`receipts[0].GasConsumed` should be `intrinsicGas + execution.GasConsumed`.

Reply:
fixed